release_notes.md

```diff
@@ -1,5 +1,6 @@
-### Release notes
-
-<!-- Please add your release notes in the following format:
-- My change description (#PR)
--->
\ No newline at end of file
+### Release notes
+
+<!-- Please add your release notes in the following format:
+- My change description (#PR)
+-->
+- Update Java Worker Version to [2.18.1](https://github.com/Azure/azure-functions-java-worker/releases/tag/2.18.1)
```

Comment:
Can you confirm you're not expecting to actually update the worker (as described in the PR), but simply adding the change to the release notes?

Reply:
> These PRs are automatically created by our release pipeline, I will investigate why they failed to update the props file.

I did move the location of this package ref, maybe your pipeline is not updated for the new location?